tools/Dashboard/DevHome.Dashboard/Views/DashboardView.xaml.cs

```diff
@@ -20,13 +20,18 @@
 using DevHome.Dashboard.TelemetryEvents;
 using DevHome.Dashboard.ViewModels;
 using DevHome.Telemetry;
-using Microsoft.UI.Dispatching;
+using Microsoft.UI.Input;
 using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Automation;
 using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
 using Microsoft.Windows.Widgets;
 using Microsoft.Windows.Widgets.Hosts;
 using Serilog;
+using Windows.System;
+using Windows.UI.Core;
+
+using DispatcherQueue = Microsoft.UI.Dispatching.DispatcherQueue;

```

Comment:
nit: `using Microsoft.UI.Dispatching;`, would that work instead?

Reply:
Since I'm using Windows.System.VirtualKey now and Windows.System also has a DispatcherQueue, unless I want to qualify each virtual key (which is clunky) something needs a statement like this. I don't hate switching it to Windows.System.VirtualKey, though, and leaving DispatcherQueue how it was.